pkgs/development/python-modules/cpycppyy/default.nix

```diff
@@ -0,0 +1,70 @@
+{
+  lib,
+  fetchFromGitHub,
+  cppyy-cling,
+  cppyy-backend,
+  cmake,
+  buildPythonPackage,
+  setuptools,
+  python3,
+  wheel,
+  pip,
+  stdenv,
+}:
+
+buildPythonPackage rec {
+  pname = "CPyCppyy";
+  version = "1.13.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "wlav";
+    repo = pname;
+    rev = "8bd9d9e10d658a33294a4ba9b2251bf1f9223469";
+    hash = "sha256-IEwdJSXG6diz7j2N7WQys5na7+xj1CYu7J3FCkW9/AQ=";
+  };
```

Comment:
```suggestion
  src = fetchFromGitHub {
    owner = "wlav";
    repo = "CPyCppyy";
    tag = "CPyCppyy-${version}";
    hash = "sha256-IEwdJSXG6diz7j2N7WQys5na7+xj1CYu7J3FCkW9/AQ=";
  };
```

Reply:
This one specifically, there's a reason for:  The current release has an issue with its version requirements, see: https://github.com/wlav/CPyCppyy/commit/8bd9d9e10d658a33294a4ba9b2251bf1f9223469

Reply:
On top of this, trying a build with that specific tag means that it will error out requiring different versions of the cppyy dependencies.